components/AppHeader.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <header class="py-3">
+        <div class="container  flex items-center justify-between">
+            <nuxt-link to="/">
+                <img src="/app-logo.svg" alt="app-logo">
+            </nuxt-link>
+            <div class="py-4 hidden lg:inline">
+                <ul class="flex items-center justify-center gap-7">
+                    <li>
+                        <nuxt-link
+                            class="text-white text-[18px] font-semibold hover:text-primary transition duration-200 ease-linear"
+                            to="/">
+                            Il progetto
+                        </nuxt-link>
+                    </li>
+                    <li>
+                        <nuxt-link
+                            class="text-white text-[18px] font-medium hover:text-primary transition duration-200 ease-linear"
+                            to="/">
+                            Perchè
+                        </nuxt-link>
+                    </li>
+                    <li>
+                        <nuxt-link
+                            class="text-white text-[18px] font-medium hover:text-primary transition duration-200 ease-linear"
+                            to="/">
+                            Vantaggi
+                        </nuxt-link>
+                    </li>
+                    <li>
+                        <nuxt-link
+                            class="text-white text-[18px] font-medium hover:text-primary transition duration-200 ease-linear"
+                            to="/">
+                            Cosa chiediamo
+                        </nuxt-link>
+                    </li>
+                    <li>
+                        <nuxt-link
+                            class="text-white text-[18px] font-medium hover:text-primary transition duration-200 ease-linear"
```

Comment:
You repeat all these classes 5 times, maybe make sense to create a component with tailwindcss